src/MetaModels/Helper/ToolboxFile.php

```diff
@@ -924,6 +926,16 @@ private function processFile($fileName)
             $information['src'] = $fileName;
         }
 
+        // Prepare the picture for provide the image size.
+        if ($information['isPicture'] = (bool) ((int) $this->resizeImages[2])) {
+            $picture = Picture::create($file, $this->getResizeImages())->getTemplateData();
+
+            $picture['alt']   = $altText;
+            $picture['title'] = $title;
```

Comment:
Don't we need more meta data here?

Reply:
Which data from the view is still needed here. All data is available that the template picture_default needs.

Reply:
The template also reads the keys `class` and `attributes`.
However, I have no idead if we want to provide them (let alone what to put for them).